### PR TITLE
Bootstrap status banner with link and button

### PR DESCRIPTION
**Pull request by thallada**
I got bootstrap styling to work and added a paragon button.

When studio is down:

screenshot-20170908152942-614x287

When you need to log in to studio:

screenshot-20170908152824-613x286

And the alert disappears if the ping returns 200.

Clicking the refresh button will re-ping Studio, and the magic of React will make the banner update automatically depending on the result!

This PR depends on paragon PR: https://github.com/edx/paragon/pull/31 please review that too if you have time, thanks!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,9 +11,8 @@
   },
   "license": "AGPL-3.0",
   "dependencies": {
-    "bootstrap": "^4.0.0-beta",
-    "edx-bootstrap": "^0.1.7",
-    "paragon": "edx/paragon",
+    "classnames": "^2.2.5",
+    "paragon": "edx/paragon#thallada/edx-bootstrap",
```
Comment (thallada) [inline package.json]: I'll change this back to master once the PR is merged. Actually, I probably should make this an NPM link...

File: src/BackendStatusBanner/statusMap.json [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+  "500": {
+    "message": "Server error from Studio",
+    "alertLevel": "danger",
+    "link": null
+  },
+  "404": {
+    "message": "You are not logged in to Studio.",
+    "alertLevel": "warning",
+    "link": {
+      "url": "http://localhost:18010/signin",
```
Comment (efischer19) [inline src/BackendStatusBanner/statusMap.json]: This is nice

File: src/BackendStatusBanner/index.jsx [hunk 3]
```diff
@@ -17,12 +20,40 @@ class BackendStatusBanner extends React.Component {
...
-          {statusMap[this.props.connectionStatus]}
+        <div
+          className={classNames(
+            styles['api-error'],
+            styles.alert,
+            styles[`alert-${status.alertLevel}`],
+          )}
+        >
+          <Button
+            display="↻"
```
Comment (efischer19) [inline src/BackendStatusBanner/index.jsx]: Is this a unicode character that we can use for a button logo? Cool.

Comment (efischer19) [reply src/BackendStatusBanner/index.jsx]: ❤️ how easy this was to declare

**Comment by efischer19** [comment]
Oh, travis failures too, looks like a simple linter catch:

```
/home/travis/build/edx/studio-frontend/src/data/actions/assets.js
  10:14  error  'getAssets' is already defined  no-redeclare
  15:34  error  'pingResponse' is not defined   no-undef
```

File: src/BackendStatusBanner/index.jsx (hunk 3 above)
Comment (thallada) [reply src/BackendStatusBanner/index.jsx]: Yup! I figured that for our dev environments a simple unicode character would do.

**Pull request merged**